### DISCUSSION TITLE
Make a program that actually works?

### DISCUSSION THREAD

**Discussion by techjunky90** [KRTirtho/spotube#928] [Q&A]
Have been following this project for about a year & there has been absolutely no progress. Every time an update is pushed, the program becomes even worse. Program has had memory leaks for months now (will literally eat through gb after gb of ram on Windows if left open for any reasonable amount of time) & there have been no efforts to fix that. Latest update breaks login. Do you even bother to test this crap before you push it to the public? I highly doubt you do. Spend less time replying to the concerns of people. Spend less time with feature request. Spend more time making an actual stable program that is usable because this crap has not been stable or usable at any point in 2023.

**Comment by KRTirtho** [comment]
@techjunky90 since you're the CEO of Useless People, Inc. why don't you help out the project by contributing instead of talking rubbish?

**Comment by techjunky90** [comment]
CEO of useless people would be you bud, look at where you stay. Bangladesh, a country of nothing more than worthless. Why would I contribute to a project that has made absolutely no progress within the last 365 days? Why would I contribute to a project that does not function properly? You literally have no idea what you are doing. Program has had memory leaks for over a year & you've made no effort to fix them because you're clueless & do not know how to fix them. Instead you focus on the meaningless things that do not bring stability or reliability. What is even funnier is, you open sourced this, you're not the only person contributing, yet you are the only person begging for donations! You're a clown!

**Reply by KRTirtho** [reply]
First of all, by contributing I meant code contribution not donations.

Now that's out of the way, nice going with disrespecting my country. At least, I can tell I'm a Bangladeshi with pride. And you can't even mention your country.

> Why would I contribute to a project that does not function properly?

Just because it's not working for you doesn't mean it isn't working for everyone. If it was unusable, the issues would've been flooded. But it's not.

> Why would I contribute to a project that does not function properly?

You're saying it's not working. That means you know how it should work. So why don't you put your knowledge into making this crap software work properly? (Or, you can't?)

> Program has had memory leaks for over a year & you've made no effort to fix them because you're clueless & do not know how to fix them

Uhm, were you living under a rock? Those were fixed almost a year ago (#599 #710) and has been released already. Yeah, it did take a month or two to fix. But in the end, it's fixed.

> Instead you focus on the meaningless things that do not bring stability or reliability

So you're telling every feature requests submitted to this project is useless and meaningless? You're telling thousands of creative people and actual contributor's vision useless? I don't see the point here. 

Look, I don't have time to talk rubbish with you. It's an Open Source project, use it or leave it. I don't care, and nobody else cares. It's your personal decision. If you can't help the project (with code), at least don't waste time of it. Nobody here is obliged to make a heaven for you (for free).

Bye, bye MR. CEO of Useless people, Inc.
